src/screens/addContact/AddContact.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {View, Text, TextInput, StyleSheet} from 'react-native';
+import {Formik} from 'formik';
+import * as Yup from 'yup';
+import CustomButton from '../../components/CustomButton';
+import FormInput from '../../common/components/FormInput';
+
+const initialValues = {
+  name: '',
+  phone: '',
+  email: '',
+  dob: '',
+};
+
+const validationSchema = Yup.object().shape({
+  name: Yup.string().required('Name is required'),
+  phone: Yup.string()
+    .matches(/^[0-9]+$/, 'Must be only digits')
+    .min(11, 'Must be exactly 11 digits')
+    .max(11, 'Must be exactly 11 digits')
+    .required('Phone number is required'),
+  email: Yup.string().email('Invalid email').required('Email is required'),
+  dob: Yup.string().required('Date of Birth is required'),
+});
+
+const AddContact = () => {
+  const handleAddContact = () => {
+    // Handle adding the contact to the database or storage here
```

Comment:
Always try to start these kinds of comments with word `TODO:`, As it would be lot easier for other developers to recognize that it is a to-do not a working code.